arho_feature_template/utils/misc_utils.py

```diff
@@ -128,3 +128,11 @@ def deserialize_localized_text(text_value: dict[str, str] | None | Any) -> str |
     if isinstance(text_value, dict):
         text = text_value.get(LANGUAGE)
     return text
+
+
+def use_wait_cursor(func):
+    def wrapper(*args, **kwargs):
```

Comment:
Dekoraattorillahan tää kannattikin tehdä, hyvä!
Tässä voisi vielä hyvin tapojen mukaisesti käyttää `functools.wraps` metodia, niin dekoraattoria käyttävien funktioiden signature pysyy samana.
```suggestion
from functools import wraps
def use_wait_cursor(func):
    @wraps(func)
    def wrapper(*args, **kwargs):
```